packages/lib/src/vis/shared/PanZoomMesh.tsx

```diff
@@ -30,12 +30,12 @@ function PanZoomMesh() {
         abscissaScale.invert(x),
         ordinateScale.invert(y)
       );
-      const { position, projectionMatrixInverse } = camera;
+      const { position } = camera;
 
-      // Project from normalized camera space (-1, -1) to (1, 1) to local camera space (-Xbound, -Ybound) to (Xbound, Ybound)
-      const cameraLocalBounds = CAMERA_TOP_RIGHT.clone().applyMatrix4(
-        projectionMatrixInverse
-      );
+      // Unproject from normalized camera space (-1, -1) to (1, 1) to world space and subtract camera position to get bounds
+      const cameraLocalBounds = CAMERA_TOP_RIGHT.clone()
+        .unproject(camera)
+        .sub(position);
```

Comment:
Because the `scale` of the camera changes, the point needs to be unprojected in the world space. The bounds are then computed by removing the position offset.